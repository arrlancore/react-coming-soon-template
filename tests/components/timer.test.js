
import React from 'react';
import renderer from 'react-test-renderer';
import Timer from '../../src/components/timer';

describe('Timer', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Timer />).toJSON()
    expect(tree).toMatchSnapshot()
  });
})
