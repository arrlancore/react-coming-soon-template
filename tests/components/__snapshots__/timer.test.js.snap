// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timer renders correctly 1`] = `
<div
  className="Countdown"
>
  <span
    className="Countdown-col"
  >
    <span
      className="Countdown-col-element"
    >
      <strong>
        01
      </strong>
      <span>
        Day
      </span>
    </span>
  </span>
  <span
    className="Countdown-col"
  >
    <span
      className="Countdown-col-element"
    >
      <strong>
        00
      </strong>
      <span>
        Hours
      </span>
    </span>
  </span>
  <span
    className="Countdown-col"
  >
    <span
      className="Countdown-col-element"
    >
      <strong>
        00
      </strong>
      <span>
        Min
      </span>
    </span>
  </span>
  <span
    className="Countdown-col"
  >
    <span
      className="Countdown-col-element"
    >
      <strong>
        00
      </strong>
      <span>
        Sec
      </span>
    </span>
  </span>
</div>
`;

exports[`Timer renders correctly for a countdown to a date 1 day in the future 1`] = `
<div
  className="Countdown"
>
  <span
    className="Countdown-col"
  >
    <span
      className="Countdown-col-element"
    >
      <strong>
        01
      </strong>
      <span>
        Day
      </span>
    </span>
  </span>
  <span
    className="Countdown-col"
  >
    <span
      className="Countdown-col-element"
    >
      <strong>
        00
      </strong>
      <span>
        Hours
      </span>
    </span>
  </span>
  <span
    className="Countdown-col"
  >
    <span
      className="Countdown-col-element"
    >
      <strong>
        00
      </strong>
      <span>
        Min
      </span>
    </span>
  </span>
  <span
    className="Countdown-col"
  >
    <span
      className="Countdown-col-element"
    >
      <strong>
        00
      </strong>
      <span>
        Sec
      </span>
    </span>
  </span>
</div>
`;

exports[`Timer renders correctly for a date in the past 1`] = `
<div
  className="Countdown"
>
  <span
    className="Countdown-col"
  >
    <span
      className="Countdown-col-element"
    >
      <strong>
        00
      </strong>
      <span>
        Days
      </span>
    </span>
  </span>
  <span
    className="Countdown-col"
  >
    <span
      className="Countdown-col-element"
    >
      <strong>
        00
      </strong>
      <span>
        Hours
      </span>
    </span>
  </span>
  <span
    className="Countdown-col"
  >
    <span
      className="Countdown-col-element"
    >
      <strong>
        00
      </strong>
      <span>
        Min
      </span>
    </span>
  </span>
  <span
    className="Countdown-col"
  >
    <span
      className="Countdown-col-element"
    >
      <strong>
        00
      </strong>
      <span>
        Sec
      </span>
    </span>
  </span>
</div>
`;